release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -320,6 +339,52 @@ The Node Feature Discovery (NFD) Operator detects hardware features available on
 
 - `cstate status`: If the `intel_idle` driver has C-states or idle states, the `cstate status` label is `true`. Otherwise, it is `false`.
 
+[id="ocp-4-8-nodes-no-reboot-cert"]
```

Comment:
I added the following note twice, once in Machine API and here in Nodes, as there was a issue for both teams to address this change, and the change does impact both components. LMK if this is a bad idea.